README.md

```diff
@@ -16,6 +16,7 @@ pnpm add -D eslint @globalbrain/eslint-config
 
 ```js
 import globalbrain from '@globalbrain/eslint-config'
+
```

Comment:
Oops, this blank line comes from the lint command. Caused by the newly added `import/newline-after-import` rule.
https://github.com/antfu/eslint-config/blob/de78b19ab28042be656b13fc2b5e055b87b0f7ea/src/configs/imports.ts#L30